mlflow/tracking/client.py

```diff
@@ -400,6 +401,7 @@ def print_experiment_info(experiments):
             view_type=view_type, max_results=max_results, page_token=page_token
         )
 
+    @experimental
```

Comment:
Should we also mark `{SqlAlchemyStore,RestStore,FileStore}.search_experiments` as experimental? MLflow users normally don't touch them.